frontend/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -42,7 +42,7 @@ interface Dimensions {
 // Use d3Graphviz in a dummy expression so the library actually gets loaded.
 // This way it registers itself in d3 as a plugin at this point.
 const _dummy_graphviz = graphviz
-_dummy_graphviz // eslint-disable-line no-unused-expressions
+_dummy_graphviz // eslint-disable-line @typescript-eslint/no-unused-expressions
```

Comment:
Looks like this rule isn't included as part of the `@typescript-eslint/recommended` plugin 😢  https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin